custom_components/elasticsearch/const.py

```diff
@@ -69,3 +67,18 @@
         }
```

Comment:
This contains privileges that don't work in serverless:
> 2024-06-07 17:38:42.322 WARNING (MainThread) [py.warnings] /workspaces/homeassistant-elasticsearch/.venv/lib/python3.12/site-packages/elasticsearch7/connection/base.py:200: ElasticsearchWarning: HasPrivileges request includes privileges for features not available in serverless mode; you will not have access to these features regardless of your permissions; cluster privileges: [manage_ilm];
  warnings.warn(message, category=ElasticsearchWarning)